### DISCUSSION TITLE
WebDriver instance

### DISCUSSION THREAD

**Discussion by jazzboy65** [selenide/selenide#2118] [Q&A] [answered]
Hello everyone!
I ran into a problem, when running autotests in Gitlab, the runner does not have time to get the webdriver instance and thus the tests fall, is it possible to check the creation of the webdriver instance using Selenide in advance?

**Comment by asolntsev** [comment]
@jazzboy65 Hi. Sorry, I didn't understand what means "to check the creation of the webdriver instance using Selenide"? 
In Selenide, you can call method `Selenide.open()` without parameters - it will open an empty browser. Is it what you meant?

**Reply by jazzboy65** [reply]
catch a mistake noSuchSessionException

**Reply by jazzboy65** [reply]
[ERROR] invalid session id
[ERROR] Build info: version: '3.12.0', revision: '7c6e0b3', time: '2018-05-08T14:04:26.12Z'
[ERROR] System info: host: 'runner-zzudqbxh-project-3561-concurrent-0', ip: '172.17.0.2', os.name: 'Linux', os.arch: 'amd64', os.version: '4.18.0-348.12.2.el8_5.x86_64', java.version: '1.8.0_342'
[ERROR] Driver info: org.openqa.selenium.chrome.ChromeDriver
[ERROR] Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 107.0.5304.87, chrome: {chromedriverVersion: 107.0.5304.62 (1eec40d3a576..., userDataDir: /tmp/.com.google.Chrome.1CAElN}, goog:chromeOptions: {debuggerAddress: localhost:39029}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: LINUX, platformName: LINUX, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}

**Reply by jazzboy65** [reply]
it seems that it does not have time to get an instance of the webdriver and thereby falls on an error, it starts every other time

**Reply by jazzboy65** [reply]
does not have time to reach open

**Comment by jazzboy65** [comment, marked as answer]
dealt with the problem selenide had nothing to do with it, sorry for the flood)
